manifests/init.pp

```diff
@@ -59,6 +59,8 @@
   Stdlib::Filemode     $config_dir_mode  = '0755',
   Stdlib::Filemode     $config_file_mode = '0644',
 
+  Optional[Array[Stdlib::Absolutepath]] $include_external_conf_files = undef,
```

Comment:
I think it would be cleaner to default to an empty array
```suggestion
  [Array[Stdlib::Absolutepath] $include_external_conf_files = [],
```

Reply:
Ok